scripts/fusion_pipeline/2_2_fusion.py

```diff
@@ -0,0 +1,89 @@
+"""Fusion Part 2.2: Calculate Distance Field on Scaled Images."""
+
+##################################################################################################
+#                                           IMPORTS
+##################################################################################################
+import argparse
+import time
+
+import dask
+import dask.array as da
+import numpy as np
+import zarr
+
+from skeleplex.skeleton.distance_field import local_normalized_distance_gpu
+
+parser = argparse.ArgumentParser()
```

Comment:
I would move this down to line 35 where you are using `parser`. I think it is helpful to group the parts of the code that are used together